<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>                                   检测区域</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>20</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>         暂无图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开相机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>采集</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>520</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>35</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>学号：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>520</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>520</y>
     <width>72</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>错误信息：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button4">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>420</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>检测采集</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>350</y>
     <width>40</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>350</y>
     <width>35</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>序号</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
